src/index.js

```diff
@@ -3,11 +3,20 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App/App';
 import reportWebVitals from './reportWebVitals';
+import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client';
+import { BrowserRouter } from "react-router-dom";
+
+const client = new ApolloClient({
+  uri: 'https://tomo-api.herokuapp.com/graphql',
```

Comment:
You could consider storing this in an env var. and use something like dotenv (which serves the same purpose as figaro in a rails app).

Reply:
If you want this to change based on environment